src/status_emitter/text.rs

```diff
@@ -343,9 +343,12 @@ impl Text {
 
 impl From<Format> for Text {
     fn from(format: Format) -> Self {
+        // Not sure what to do here. The architecture allows for multiple emitter
+        // implementations, but the code assumes that there is only implementation,
+        // namely Text.
```

Comment:
yea this impl should be removed. Probably just in favor of an `From<Format> for Arc<dyn StatusEmitter>` impl or just some free function

Reply:
OK, I will remove it.

Reply:
Done.